README.md

```diff
@@ -136,20 +136,28 @@ We currently use [clippy](https://github.com/rust-lang-nursery/rust-clippy) to e
 This check is run automatically during CI builds, and in a `pre-commit`
 hook. You can run do a clippy check with `./scripts/run_clippy.sh`.
 
-## Development cluster
+## Running TestNet locally
 
-To spin multiple nodes, you must first spin up one node and then use it as boot node for all the rest:
+Navigate to the root of the project. Make sure we are not re-using existing storage:
```

Comment:
this probably should be a side note "To restart the network", because in a real situation you actually want to continue running.

Reply:
For setup, also need to call keygen for alice & bob

Reply:
Added note.

We should add the flag later, currently it is outside the scope of the PR.

Upon the start, the client will generate and save the keystore if it does not find one. See:
https://github.com/nearprotocol/nearcore/blob/master/core/primitives/src/signer.rs#L58